projects/spectator/test/spy-object/person.ts

```diff
@@ -6,6 +6,18 @@ export class Person {
     return 'Hi!';
   }
 
+  public saySomething(something: string): string {
```

Comment:
Let's use https://github.com/tsdjs/tsd to test the types

Reply:
It's a little less "good", but I could also use `// @ts-expect-error` and add fail tests

Reply:
I'm fine with @ts-expect-error

Reply:
I've added additional tests decorated with `@ts-expect-error`